common.blocks/sidebar/__sorting/sidebar__sorting.js

```diff
@@ -25,13 +25,16 @@ modules.define(
             window.location.reload();
         },
 
-        _getParams(sort, direction) {
-            const uri = Location.getUri();
-            const query = uri.queryParams;
-            const params = {};
+        _getParams: function(sort, direction) {
+            var uri = Location.getUri();
+            var query = uri.queryParams;
+            var params = {};
+            var exclude_params = ['pagination'];
```

Comment:
2. thanks, that is really good idea!
3. yes, finally i did like that - but now we always receive pagination=undefined as a query param, even if we didn't have it before